test/CheckoutSdk.Tests/ValidationTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Checkout.Sdk.Payments;
+using Shouldly;
+using Xunit;
+
+namespace Checkout.Sdk.Tests
+{
+    public class ValidationTests
+    {
+        [Theory]
+        [InlineData("")]
+        [InlineData("test")]
+        public void CustomerSource_ThrowExceptionIfSourceEmailDoesNotHaveAtSign(string email)
```

Comment:
We tend to use the BDD style syntax for test naming e.g. `GivenEmailInvalidShouldThrowFormatException`